consult.el

```diff
@@ -932,6 +940,81 @@ Depending on the selected item OPEN-BUFFER, OPEN-FILE or OPEN-BOOKMARK will be u
   (interactive)
   (consult--buffer #'switch-to-buffer #'find-file #'bookmark-jump))
 
+;;;###autoload
+(defun consult-kmacro (arg)
+  "Run a chosen keyboard macro.  With prefix ARG, run the macro that many times.
+
+Macros containing mouse clicks can't be displayed properly.  To
+keep things simple, macros with an empty display string (e.g.,
```

Comment:
Shorter description please. Remove clicks since they are not displayed properly or sth?